src/OpenTelemetry/Trace/SpanSdk.cs

```diff
@@ -827,7 +827,26 @@ private bool IsAttributeValueTypeSupported(object attributeValue)
                 return true;
             }
 
-            // TODO add array support
+            if (attributeValue is Array array)
```

Comment:
Does it work for `IEnumerable`? Do you think we should support them?

Reply:
Added support for `IEnumerable`